README.md

```diff
@@ -11,24 +11,84 @@ This project is for OSIS which integrates [Scality RING](https://www.scality.com
 
 ## Running the Application
 
-### Command to run Docker Image
-A user needs to be logged into docker using URL: registry.scality.com.
-```sh
-    $ docker login -u <username in registry.scality.com> registry.scality.com
-```
-Once logged images can be pulled and run locally from either dev namespace using the hash or production namespace.
+### Commands to run Docker Image
+1. Docker images can be built locally or pulled locally from either dev namespace using the hash or production namespace.
+
+    Local build:
+    ```sh
+        $ docker build -t <local_image_name> .
+    ```
+    Dev:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
+    ```
+    Production image:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality/vmware-ose-scality:<tag>
+    ```
+1. Generate a self-signed SSL certificate and store it in a binary PKCS#12 format file with extension `.p12` file. (Refer [here](#To-generate-PKCS12-file-for-self-signed-SSL-certificate))
+
+1. Create an environment variables file `application.properties`
+    - Sample `application.properties` file can be found [here](src/main/resources/application.properties).
+    - Update `server.ssl.key-store-password` and `server.ssl.key-alias` with key-store password and key-store alias of the `.p12` file respectively.
+
+1. To run the docker image locally using local build image or from either dev namespace using the hash or production namespace with the environment file `application.properties`
+
+   Local build image:
+    ```sh
+     $ docker run \
+       --env-file application.properties \
+       -it \
+       -p 8443:8443 \
+       --mount type=bind,source=<absolute_path_to_.p12_file>,target=/app/lib/osis.p12 \
+       <local_image_name>
+    ```
+   Dev:
+    ```sh
+     $ docker run \
+       --env-file application.properties \
+       -it \
+       -p 8443:8443 \
+       --mount type=bind,source=<absolute_path_to_.p12_file>,target=/app/lib/osis.p12 \
+       registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
```

Comment:
I don't believe our registry has public read access or any other way to browse available images unlike docker hub. Ideally a short sentence of something like: "You can also use our public registry to run by viewing available images here" with a link to something like registry.scality.com/browse but that doesn't exist. @tcarmet does Harbor provide read-only views of public namespaces? So we could link something like the above?

I just wouldn't repeat the same command 3 times. Plus, it would be nice to provide a means of finding out what the latest images are.